llama-rs/src/loader2.rs

```diff
@@ -190,7 +196,7 @@ pub(crate) fn load(
 
     (load_progress_callback)(LoadProgress::PartLoaded {
         file: &path,
-        byte_size: 0,
+        byte_size: filesize,
```

Comment:
I guess the argument for leaving this 0 is that we don't know the exact size of the tensors since there's other stuff in the file too.  It seems reasonable to fall back to the whole file in that case, but the other decent option would be to not print tensor size if we get 0 (or change the type to `Option<u64>`).